src/main/webapp/app/modules/administration/user-management/user-management-update.tsx

```diff
@@ -124,10 +146,12 @@ const mapStoreToProps = (storeState: IRootStore) => ({
   user: storeState.userStore.entity,
   loading: storeState.userStore.loading,
   updating: storeState.userStore.updating,
+  featureFlags: storeState.featureFlagStore.entities,
   getUser: storeState.userStore.getEntity,
   updateUser: storeState.userStore.updateEntity,
   createUser: storeState.userStore.createEntity,
   reset: storeState.userStore.reset,
+  getFeatureFlags: storeState.featureFlagStore.getEntities,
```

Comment:
Add a prop here: `isDev: hasAnyAuthority(authStore.account.authorities, [AUTHORITIES.DEV])` to determine if user is a dev instead of the isDev state variable